simulators/nav/src/components/NavSimulator.vue

```diff
@@ -440,6 +448,7 @@ export default class NavSimulator extends Vue {
       /* Subscriptions */
       [
         { topic: '/autonomous',     type: 'Joystick' },
+        { topic: '/auton_enabled',  type: 'Enable' },
```

Comment:
probably also add the teleop_enabled? @ankithu?

Reply:
not sure it really needs to subscribe to anything

Reply:
Just reverted back to only publish enable